docs/ADDRESS_LOOKUP.md

```diff
@@ -11,7 +11,63 @@
 - If you're integrating with Drop-in:
     - Implement the mandatory `onAddressLookupQueryChanged(query: String)` callback and optional `onAddressLookupCompletion(lookupAddress: LookupAddress)` callback.
     - Pass the result of these actions by using `AddressLookupDropInServiceResult` class.
+      ```kotlin
+      override fun onAddressLookupQueryChanged(query: String) {
+          // Use the query parameter to make a call to your desired maps endpoint 
+          // Map the address objects LookupAddress object and pass it back to Drop-in as follows:
+          val options: List<LookupAddress> = // mapped list of address objects
+          sendAddressLookupResult(AddressLookupDropInServiceResult.LookupResult(options))  
+      }
+
+      override fun onAddressLookupCompletion(lookupAddress: LookupAddress): Boolean {
+          // Optionally use this callback to get the complete details for a LookupAddress option selected by the shopper
```

Comment:
Maybe you can reduce the line size to make the code section readable without scrolling horizontally?